home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>YoutTube - Custome</title>
+</head>
+<body>
+    <ul class="header">
+        <li ><a href="#">YoutTube</a></li>
+        <li style="float: right;"><a href="#">LogOut</a></li>
+    </ul>
```

Comment:
- [ ] Kindly please make your heading menus using `nav` tag. It will make your code clean and understandable by others

Reply:
Please ensure that you phrase your comments politely, as suggestions instead of orders. For example:

- This is a polite suggestion: "I think it would be better to use semantic tags"
- This reads as an order: "Please use semantic tags"

Reply:
- [ ] Kindly I recommend you to use the `nav` tag in your heading menu.